### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by azr**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 0.0.12 (Unreleased)
```
Comment (sylviamoss) [inline CHANGELOG.md:1]:
```suggestion
## 0.0.13 (Upcoming)

## 0.0.12 (Frebruary 11, 2021)
```

Why not this way?

Comment (sylviamoss) [reply CHANGELOG.md:1]: I think the release job will look for upcoming to write the release date and get the correct tag for the next release

Comment (azr) [reply CHANGELOG.md:1]: 0.0.12 was not released, so I was planning on just moving the tag

Comment (sylviamoss) [reply CHANGELOG.md:1]: Anyways to release the 0.0.13 we need the `Upcoming` one. I think 🤔 
```suggestion
## 0.0.13 (Upcoming)

## 0.0.12 (Unreleased)
```

Comment (nywilken) [reply CHANGELOG.md:1]: That's correct the script will match against "Upcoming". Do we not have any changes? Should we add a note?

Comment (azr) [reply CHANGELOG.md:1]:
```suggestion
## 0.0.12 (Upcoming)
```

Comment (azr) [reply CHANGELOG.md:1]: Does this sound good ? I can also just go for 0.0.13

Comment (azr) [reply CHANGELOG.md:1]: Yeah, you're right, maybe let's not change the release tag

**Review by sylviamoss — APPROVED** [review]

**Pull request merged**